### DISCUSSION TITLE
Guide for new disk features in 1.9.x and 1.10.x

### DISCUSSION THREAD

**Discussion by runningman84** [siderolabs/talos#10071] [Q&A]
As many users with a single disk setup I am asking myself what would be the correct configuration to ensure that longhorn and the local-path-provisioner get an own partition? The general idea would look like this:

- system (whatever is needed)
- state (whatever is needed)
- ephemeral (use all remaining disk space) 
- longhorn (min size 10G, max size 200G) /var/lib/longhorn
- local-path(min size 10G, max size 100G)  /var/lib/local-path-provisioner

https://github.com/siderolabs/talos/issues/8367

**Comment by smira** [comment]
This should be hopefully finally available for Talos 1.10, right now there's no good answer.

**Comment by runningman84** [comment]
With talos 1.11 that setup should be possible, do you have any example in the docs?

**Reply by smira** [reply]
Just follow the [documentation on disk management](https://www.talos.dev/v1.11/talos-guides/configuration/disk-management/).
